src/Backend/Modules/Analytics/Tests/GoogleClient/ConnectorTest.php

```diff
@@ -3,7 +3,6 @@
 namespace Backend\Modules\Analytics\Tests\GoogleClient;
 
 use Backend\Modules\Analytics\GoogleClient\Connector;
-use Common\ModulesSettings;
```

Comment:
zou het niet een betere fix zijn om in de `getModulesSettingsMock` method `ModulesSettings::class` te gebruiken?